test/jdk/tools/jpackage/helpers/jdk/jpackage/test/AdditionalLauncher.java

```diff
@@ -81,12 +81,14 @@ public AdditionalLauncher addRawProperties(
         return this;
     }
 
-    public AdditionalLauncher setShortcuts(boolean menu, boolean desktop) {
+    public AdditionalLauncher setShortcuts(boolean menu, boolean shortcut) {
+        withMenuShortcut = menu;
+        withShortcut = shortcut;
         if (TKit.isLinux()) {
```

Comment:
I'd move add `addRawProperties()` calls to `initialize()` method. This will prevent multiple `addRawProperties()` calls for "linux-shortcut" and "win-shortcut" properties in case `setShortcuts()` method is called multiple times.